releases/release-7.6.0.md

```diff
@@ -0,0 +1,288 @@
+
+7.6.0 introduces the following key features and improvements:
+
+<table>
+<thead>
+  <tr>
+    <th>Category</th>
+    <th>Feature/Enhancement</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="3">Scalability and Performance<br></td>
+    <td><a href="https://docs.pingcap.com/tidb/v7.6/sql-plan-management#universal-binding">Universal SQL binding</a> {/* tw@Oreoxmt */}</td>
+    <td>In use cases with hundreds or thousands of same-schema databases -- such as SaaS data platforms storing a database per customer with equivalent business logic -- having up-to-date stats at all times is challenging. For this, SQL bindings can be used to lock in performance. However, doing so across so many tables is infeasible. Therefore, universal SQL bindings allow for broadcasting bindings across all schema-equivalent databases.</td>
+  </tr>
+  <tr>
+    <td><a href="https://docs.pingcap.com/tidb/v7.6/br-snapshot-guide#restore-cluster-snapshots">Up to 10x acceleration to snapshot restore</a> {/* tw@Oreoxmt */}</td>
+    <td>A new 2-phase region scatter algorithm for preparing a snapshot restore to a cluster was introduced. In clusters with many TiKV nodes, this dramatically improves the cluster resource efficiency by more evenly distributing loading across the nodes, more effectively using per-node network bandwidth. In several real world cases, this causes a 1,000% acceleration of restore speeds.</td>
+  </tr>
+  <tr>
+    <td><a href="">Achieve up to 10 times faster for creating tables in batch</a>  {/* tw@hfxsd */}</td>
+    <td>With the implementation of the new DDL architecture in v7.6.0, the performance of batch table creation has witnessed a remarkable improvement, up to 10 times faster. This substantial enhancement drastically reduces the time needed for creating numerous tables. This acceleration is particularly noteworthy in SaaS scenarios, where the prevalence of high volumes of tables, ranging from tens to hundreds of thousands, is a common challenge.</td>
+  </tr>
+  <tr>
+    <td rowspan="2">Reliability and Availability<br></td>
+    <td><a href="https://docs.pingcap.com/tidb/v7.6/tiproxy/tiproxy-overview">TiProxy support</a>  {/* tw@ran-huang */}</td>
+    <td>Full support for the TiProxy service, easily deployable via deployment tooling, to manage and maintain connections to TiDB so that they live through rolling restarts, upgrades, or scaling events.</td>
+  </tr>
+  <tr>
+    <td><a href="">Data Migration (DM) officially supports MySQL 8.0 (GA)</a>  {/* tw@hfxsd */}</td>
+    <td>In previous migration scenarios from MySQL 8.0 with numerous tables using DM, extensive manual modification of table creation statements was required. Starting from v7.6.0, you can directly configure alternative charsets and collations in DM. DM seamlessly manages these adjustments in the downstream, ensuring a much smoother migration process from MySQL 8.0.</td>
```

Comment:
@Frank945946 PTAL

Reply:
Previously, DM migration for MySQL 8.0 was only an experimental feature and not suitable for production environments. TiDB v7.6.0 has improved the stability and compatibility of this feature, allowing you to smoothly and rapidly migrate data from MySQL 8.0 to TiDB in a production environment. 
@samhld The mention of 'Starting from v7.6.0, you can directly configure alternative charsets and collations in DM' has been removed as this feature has been postponed to the next version.